src/frontend/App/hooks/useHentKontantstøtteUtbetalinger.ts

```diff
@@ -0,0 +1,41 @@
+import { Ressurs, RessursStatus } from '../typer/ressurs';
+import { useApp } from '../context/AppContext';
+import { useEffect, useState } from 'react';
+import { AxiosRequestConfig } from 'axios';
+
+export enum EFinnesKontantstøtteUtbetaling {
+    JA = 'JA',
+    NEI = 'NEI',
+    UKJENT = 'UKJENT',
+}
+
+type IKontantstøtteUtbetaling = {
```

Comment:
```suggestion
type KontantstøtteUtbetaling = {
```
Av samme grunn som over ville jeg fjernet I. Spesielt siden dette ikke er et interface også